src/routers/links.py

```diff
@@ -11,16 +15,22 @@
 )
 
 
-@router.get("/", response_model=List[schema.Link])
-def get_links(db: Session = Depends(get_db)):
-    return crud.get_links(db=db)
+@router.get("/", response_model=List[LinkSchema])
+def get_links(models_manager: ModelsManager = models_manager_dependency):
+    return models_manager.links.all()
+
+
+@router.get("/{link_id}", response_model=LinkSchema)
+def get_link(link_id: int, models_manager: ModelsManager = models_manager_dependency):
+    return models_manager.links.get(link_id)
 
 
-@router.post("/", response_model=schema.Link, status_code=status.HTTP_201_CREATED)
-def create_link(link: schema.LinkCreate, db: Session = Depends(get_db)):
-    return crud.create_link(db=db, link=link)
+@router.post("/", response_model=LinkSchema, status_code=status.HTTP_201_CREATED)
+def create_link(link: LinkCreateSchema, models_manager: ModelsManager = models_manager_dependency):
+    return models_manager.links.create(model_schema=link)
 
 
-@router.post("/{link_id}/set_labels", response_model=schema.Link)
-def set_link_labels(link_id: int, labels_ids: List[int], db: Session = Depends(get_db)):
-    return crud.set_link_labels(db=db, link_id=link_id, labels_ids=labels_ids)
+@router.post("/{link_id}/set_labels", response_model=LinkSchema)
+def set_link_labels(link_id: int, labels_ids: List[int], models_manager: ModelsManager = models_manager_dependency):
+    labels = models_manager.labels.filter_by_ids(ids=labels_ids)
```

Comment:
lets rename to just 
filter_by(ids=sdfsdfsdf)

more fluent in english read and extendable in future
or did u name it like that cuz of collision?
this is y im not sure about the get attr thing

Reply:
I chsoe this name because it required the usage of `in_` operator 
filter_by is used with single value